src/components/Accordion/Accordion.style.ts

```diff
@@ -0,0 +1,120 @@
+import { rem } from 'polished';
+import styled, { css } from 'styled-components';
+
+import { ChevronDown, CircleWarning } from '../../icons';
+import { Paragraph } from '../../typography';
+
+import { grayscale } from '../../color';
+import { core } from '../../tokens';
+
+export const AccordionStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${rem(8)};
+`;
+
+export const Wrapper = styled.div<{
+  format: 'basic' | 'secondary';
+  active: boolean;
+  disabled: boolean;
+}>`
+  ${({ disabled }) =>
+    disabled &&
+    css`
+      pointer-events: none;
+      opacity: 0.4;
+    `}
+  ${({ active, theme, format }) =>
+    active &&
+    css`
+      background-color: ${format === 'basic'
+        ? theme.name === 'dark'
+          ? grayscale(800)
+          : grayscale(50)
+        : 'none'};
+    `}
+  border-radius: ${rem(10)};
+  color: ${core.color.text.primary};
+  ${({ active, theme, format }) =>
+    active &&
+    css`
+      border: ${format === 'secondary'
+        ? theme.name === 'dark'
+          ? `${rem(1)} solid ${grayscale(800)}`
+          : `${rem(1)} solid ${grayscale(50)}`
```

Comment:
If the border color is based on the surface token, you could hold that value in a const (or css property!). You should be able to use the `--` css custom property syntax (see example from https://styled-components.com/)